guides/README.md

```diff
@@ -91,16 +91,16 @@ Please read these guidelines before opening a Pull Request. For more information
 The content in this repository is shared between the upstream Foreman community and branded downstream products such as Red Hat Satellite and orcharhino by ATIX.
 Therefore, never write "Foreman", "Satellite", or "orcharhino" words directly, but use the following variables:
 
-| Variable | Upstream value | Downstream value |
-| -------- | -------------- | ---------------- |
-| {ProjectNameXY} | Foreman 1.22 | Red Hat Satellite 6.5 |
-| {ProjectNameX} | Foreman | Red Hat Satellite 6 |
-| {ProjectName} | Foreman | Red Hat Satellite |
-| {ProjectXY} | Foreman 1.22 | Satellite 6.5 |
-| {ProjectX} | Foreman | Satellite 6 |
-| {Project} | Foreman | Satellite |
-| {SmartProxyServer} | Smart Proxy server | Capsule Server |
-| {SmartProxy} | Smart Proxy | Capsule |
+| Variable | Upstream value | Downstream by Red Hat | Downstream by ATIX |
```

Comment:
Do you think we should have the corresponding versions here? 
We'll forget to update it, won't we?

Reply:
Yes! I am all for removing it.

If you want to remove the whole table, then we should extend how the attributes work. I am fine with both.

Rebased to master.